packages/components/table/src/util.ts

```diff
@@ -371,15 +377,34 @@ export function walkTreeNode(
 
 const getTableOverflowTooltipProps = (
   props: TableOverflowTooltipOptions,
-  content: string
+  innerText: string,
+  row: T,
+  column: TableColumnCtx<T>
 ) => {
+  // merge popperOptions
+  const popperOptions = {
+    strategy: 'fixed',
+    ...props.popperOptions,
+  }
+
+  const tooltipFormatterContent = isFunction(column.tooltipFormatter)
+    ? column.tooltipFormatter({ row, column, cellValue: row[column.property] })
```

Comment:
```suggestion
    ? column.tooltipFormatter({ row, column, cellValue: getProp(row, column.property).value })
```

property 可以是 `a.b` 这种形式，这里不能够直接使用 `row[column.property]`

Reply:
好，明天早上改一下

Reply:
@tolking 感谢你的指正，已修改~^^

Reply:
@zhixiaotong 

抱歉，我才注意到你是和 formatter  保持一致，或许 (row: any, column: any, cellValue: any) => VNode | string 更合适

Reply:
> @zhixiaotong
> 
> 抱歉，我才注意到你是和 formatter 保持一致，或许 (row: any, column: any, cellValue: any) => VNode | string 更合适

你指的是` (data: {row: any, column: any, cellValue: any}) => VNode | string`改成` (row: any, column: any, cellValue: any) => VNode | string`吗？

Reply:
![image](https://github.com/user-attachments/assets/29266401-75ba-4320-a1ce-dcd84c2de76e)

Reply:
保持现在这样也行

WDYT @btea

Reply:
好的~ ^^

Reply:
> WDYT @btea

保持现状就行。